### ISSUE TITLE
20NG Training Set Reported Results Question

### ISSUE THREAD

**Issue by jharrang** [Tiiiger/SGC#34]
For your 20NG results reported in [your original paper](https://arxiv.org/pdf/1902.07153.pdf) (`88.5 ± 0.1`), was the model trained on the full public 20NG `train` set, of size 11314, or were the reported results generated using the [code currently in this repo](https://github.com/Tiiiger/SGC/blob/2c7a2727e82e462d8ef9d6e57f0b08888e16488f/downstream/TextSGC/train.py#L67), which appears to exclude the validation set from the data used to train the model? (The latter would use a training set of size 10183).

Also, I notice that in Table 4 you reported a different result (`87.9 ± 0.2`) for the cited [Yao GCN](http://1.15.185.141/ir2019fall/pdf/wuhaoming.pdf) paper than they did themselves (`0.8634 ± 0.0009`). Do you know why that is the case?

Thanks!

**Comment by Tiiiger** [comment]
hi @jharrang 

sorry for a late reply. 

hoestly it has been a while that I don't remember the exact details. But I am sure that the code we used in this repo is the code we used to get the reported results. Historically, the data pipeline is taken from this repo (https://github.com/codeKgu/Text-GCN). 

and sorry I am not sure why did we get better results for their method ...
